manifest.json

```
@@ -9,7 +9,7 @@
         "es": "DokuWiki es un sistema de Wiki de uso sencillicimo y compatible con los estándares.",
         "it": "DokuWiki è un Wiki aderente agli standard, semplice da usare, finalizzato principalmente alla creazione di documentazione di qualsiasi tipo."
     },
-    "version": "2018-04-22a~ynh3",
+    "version": "2018-04-22b~ynh4",
```

```suggestion
    "version": "2018-04-22b~ynh1",
```
If the app version is changed, set the package version to 1 ;-)

---

I wasn't aware of that. I thought that the package version should "never change", only be increased. Well then :)